payments-ui-core/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodRequirements.kt

```diff
@@ -289,3 +289,9 @@ internal val BoletoRequirement = PaymentMethodRequirements(
     siRequirements = setOf(Delayed),
     confirmPMFromCustomer = true,
 )
+
+internal val KonbiniRequirement = PaymentMethodRequirements(
+    piRequirements = setOf(Delayed),
+    siRequirements = null,
+    confirmPMFromCustomer = true,
```

Comment:
I think we also don't want this to be true

Reply:
🤦 thank you